packages/admin/src/pages/build-application/[applicationId]/[sectionId]/question-type/index.page.tsx

```diff
@@ -5,44 +5,63 @@ import {
   ValidationError,
 } from 'gap-web-ui';
 import { GetServerSideProps } from 'next';
-import getConfig from 'next/config';
-import CustomLink from '../../../../components/custom-link/CustomLink';
-import Meta from '../../../../components/layout/Meta';
+import CustomLink from '../../../../../components/custom-link/CustomLink';
+import Meta from '../../../../../components/layout/Meta';
 import ResponseType, {
   ResponseTypeLabels,
-} from '../../../../enums/ResponseType';
-import { getApplicationFormSummary } from '../../../../services/ApplicationService';
-import { postQuestion } from '../../../../services/QuestionService';
+} from '../../../../../enums/ResponseType';
+import { getApplicationFormSummary } from '../../../../../services/ApplicationService';
+import { postQuestion } from '../../../../../services/QuestionService';
 import {
   addFieldsToSession,
   getSummaryFromSession,
   getValueFromSession,
-} from '../../../../services/SessionService';
-import { QuestionSummary } from '../../../../types/QuestionSummary';
-import callServiceMethod from '../../../../utils/callServiceMethod';
-import { getSessionIdFromCookies } from '../../../../utils/session';
-import QuestionTypeHint from './components/QuestionTypeHint';
-import {
-  getErrorPageParams,
-  questionErrorPageRedirect,
-} from './newQuestionServiceError';
+} from '../../../../../services/SessionService';
+import { QuestionSummary } from '../../../../../types/QuestionSummary';
+import { getSessionIdFromCookies } from '../../../../../utils/session';
+import QuestionTypeHint from '../components/QuestionTypeHint';
+import { questionErrorPageRedirect } from '../newQuestionServiceError';
+import ResponseTypeEnum from '../../../../../enums/ResponseType';
+import QuestionPageGetServerSideProps from '../../../../../utils/QuestionPageGetServerSideProps';
 
 type RequestBody = {
   responseType: ResponseType;
   _csrf?: string;
 };
 
+const redirectQuestionType = [
+  ResponseType.Dropdown,
+  ResponseType.MultipleSelection,
+  ResponseType.LongAnswer,
+];
+
 const WORD_LIMIT_MAP = {
   [ResponseType.ShortAnswer]: 300,
-  [ResponseType.LongAnswer]: 5000,
```

Comment:
not much point in having a map for this if it can only ever be one value now?

Reply:
Sure, have rm'd that